src/components/ManageAccountsDrawer/components/LedgerOpenApp.tsx

```diff
@@ -110,18 +113,42 @@ export const LedgerOpenApp = ({ chainId, onClose, onNext }: LedgerOpenAppProps)
     if (isValidApp) {
       onNext()
     } else {
-      console.error('Correct Ledger app is not open', slip44Key)
+      toast({
+        title: translate('walletProvider.ledger.errors.appNotOpen', { app: slip44Key }),
```

Comment:
There's now a discrepancy between `slip44Key` for the toast and `asset.networkName` for the openApp title - do we want to consistently use `slip44Key`?

Reply:
Ah good catch, the user doesn't care about the slip key hey. I'll update.

Reply:
Or probably the other way around - just double checked and `keyof typeof slip44Key` are UpperCamelCased, while Ledger apps are human-readable e.g `BitcoinCash` slip44Key key (the duplicate key isn't a mistake), vs. `Bitcoin Cash` app

Reply:
Though I just read `getSlip44KeyFromChainId` again and disregard all of the above, I'm a derp - both of these will be wrong either way - `getSlip44KeyFromChainId` has the *right* SLIP44 for chains (as it should).

So what we want instead is to group EVM chains into the `ethChainId` umbrella as mentioned in the previous comment, and then use `networkName` everywhere.

See how the current Ledger `<Chains />` component handles it (notice there's only an `ethChainId` option):

https://github.com/shapeshift/web/blob/a2ffbc9def5d71bb9c7f34fa23309b350d677147/src/context/WalletProvider/Ledger/components/Chains.tsx#L77-L79

Reply:
Unified EVMs under Ethereum in https://github.com/shapeshift/web/pull/6790/commits/265120d4a467506fb2c45062770e6edb78d67e01